tests/sdk/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -xe
+cd -- `dirname ${BASH_SOURCE[0]}`
+
+ROOT_DIR="../.."
+
+export SEBAK_GENESIS=GDIRF4UWPACXPPI4GW7CMTACTCNDIKJEHZK44RITZB4TD3YUM6CCVNGJ
+export SEBAK_COMMON=GDYIHSHMDXJ4MXE35N4IMNC2X3Q3F665C5EX2JWHHCUW2PCFVXIFEE2C
+
+NODE=""
+function dumpLogsAndCleanup () {
+    if [ ! -z "${NODE}" ]; then
+        docker logs ${NODE} || true
+        docker rm -f ${NODE} || true
+    fi
+}
+
+trap dumpLogsAndCleanup EXIT
+
+## Build the node runner docker image
+NODE_IMAGE=$(docker build -q \
+    --build-arg BUILD_MODE="test" \
+    --build-arg BUILD_PKG="./cmd/sebak" \
+    --build-arg BUILD_ARGS="-coverpkg=./... -tags integration -c -o /go/bin/sebak"  \
+    ${ROOT_DIR}/ | cut -d: -f2)
+
+if [ -z ${NODE_IMAGE} ]; then
+    echo "Failed to build node runner docker image" >&2
+    exit 1
+fi
+
+## Build the docker builder image
+IMAGE=$(docker build --tag sebak:builder_sdk -q \
```

Comment:
There's still a tag being done here. Remove it, and instead add an `ARG` to your runner which takes the image to copy from.

Reply:
I tried that but It is impossible to do that.
In a docker file I can not set --from=imagehash

Reply:
Change your Dockerfile to the following:
```Dockerfile
## Builder part is quite heavy as it depends on the Go toolchain
## Work around Moby bug 34482
ARG BUILDER
FROM ${BUILDER} AS builder

FROM alpine:latest
LABEL maintainer="BOSCoin Developers <devteam@boscoin.io>"

COPY --from=builder /go/bin/sdk /sebak/
COPY ./entrypoint.sh /sebak/

WORKDIR /sebak/
ENTRYPOINT [ "./entrypoint.sh" ]
CMD []
```

And apply this diff to `run.sh`:
```diff
diff --git a/tests/sdk/run.sh b/tests/sdk/run.sh
index 2519f1c..2be6077 100755
--- a/tests/sdk/run.sh
+++ b/tests/sdk/run.sh
@@ -31,7 +31,7 @@ if [ -z ${NODE_IMAGE} ]; then
 fi

 ## Build the docker builder image
-IMAGE=$(docker build --tag sebak:builder_sdk -q \
+IMAGE=$(docker build -q \
     --build-arg BUILD_MODE="install" \
     --build-arg BUILD_PKG="./tests/sdk" \
     ${ROOT_DIR}/ -f ${ROOT_DIR}/Dockerfile_client.build | cut -d: -f2)
@@ -41,7 +41,7 @@ if [ -z ${IMAGE} ]; then
     exit 1
 fi

-TEST_IMAGE=$(docker build -q \
+TEST_IMAGE=$(docker build -q --build-arg BUILDER_IMAGE=${IMAGE} \
     . | cut -d: -f2)

 if [ -z ${TEST_IMAGE} ]; then
```

Reply:
wow. I thought only `COPY` statement.
Thx!

Reply:
Yep it's a bug in Moby (docker), I was about to report it but found someone already did, and there is a workaround.

Reply:
DONE!